specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -555,6 +555,183 @@
+            "format": "date",
+            "required": true
+          },
+          {
+            "$ref": "#/parameters/WeatherDataUnit"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DailyHistoricalActualsResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/historical/records/daily/{format}": {
+      "get": {
+        "description": "**Get Daily Historical Records**\n  \n  \n**Applies to**: G1 and G2 pricing tiers.\n\n\nThe service returns climatology data such as past daily records for a given coordinate location.",
+        "operationId": "Weather_GetDailyHistoricalRecords",
+        "x-ms-examples": {
+          "DailyHistoricalRecords": {
+            "$ref": "./examples/GetDailyHistoricalRecords.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ResponseFormat"
+          },
+          {
+            "name": "query",
+            "description": "Coordinates of the location for which current conditions information is requested. The applicable query is specified as a comma separated string composed by latitude followed by longitude e.g. \"47.641268,-122.125679\".",
+            "in": "query",
+            "type": "string",
+            "required": true
+          },
+          {
+            "name": "startDate",
+            "description": "Start date in ISO 8601 format, for example, 2019-10-27.",
+            "in": "query",
+            "type": "string",
+            "format": "date",
+            "required": true
+          },
+          {
+            "name": "endDate",
+            "description": "End date in ISO 8601 format, for example, 2019-10-28.",
+            "in": "query",
+            "type": "string",
+            "format": "date",
+            "required": true
+          },
+          {
+            "$ref": "#/parameters/WeatherDataUnit"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DailyHistoricalRecordsResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/historical/normals/daily/{format}": {
+      "get": {
+        "description": "**Get Daily Historical Normals**\n  \n  \n**Applies to**: G1 and G2 pricing tiers.\n\n\nThe service returns climatology data such as past daily normals for a given coordinate location.",
```

Comment:
To provide more details about what Normals supports and how, I propose we change:

The service returns climatology data such as past daily normals for a given coordinate location.

To:

Get Daily Historical Normals service returns climatology data such as past daily normal temperatures, precipitation and cooling/heating degree day information for the day at a given coordinate location. Normals are a 30-year average for temperatures for a specific location. As is standard practice in climatology, the 30-year average covers years 1991-2020, this data will be used for one decade and then will reset in the year 2030.  Generally, historical data may be available as far back as the last 5 to 40+ years, depending on the location.